.github/workflows/codespell.yaml

```diff
@@ -0,0 +1,20 @@
+name: codespell
+on:
+  pull_request:
+    branches:
+      - master
+      - enterprise
+permissions: {}
+jobs:
+  codespell:
+    name: Check for spelling errors
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      - uses: codespell-project/actions-codespell@master
+        with:
+          ignore_words_list: "ser"
```

Comment:
YES! Thank you, it was driving me nuts ;-)

Reply:
Why do we have this word "ser" in our documents enough to drive you nuts?